utilities/xmlrpcpp/test/CMakeLists.txt

```diff
@@ -3,6 +3,19 @@ if(TARGET xmlrpcvalue_base64)
   target_link_libraries(xmlrpcvalue_base64 xmlrpcpp)
 endif()
 
+if(NOT WIN32)
+catkin_add_gtest(test_socket
+  test_socket.cpp
+  test_system_mocks.c
+  ../src/XmlRpcSocket.cpp
+  ../src/XmlRpcUtil.cpp
+)
+set_target_properties(test_socket PROPERTIES
+  LINK_FLAGS
+  "-Wl,--wrap=accept -Wl,--wrap=bind -Wl,--wrap=close -Wl,--wrap=connect -Wl,--wrap=getaddrinfo -Wl,--wrap=getsockname -Wl,--wrap=listen -Wl,--wrap=read -Wl,--wrap=setsockopt -Wl,--wrap=select -Wl,--wrap=socket -Wl,--wrap=write -Wl,--wrap=fcntl -Wl,--wrap=freeaddrinfo"
+)
```

Comment:
The block within the `if` and `endif` should be indented one level.

Reply:
Done.